src/js/extensions/anchor.js

```diff
@@ -242,7 +242,7 @@
                 return 'tel:' + value;
             } else {
                 // Check for URL scheme and default to http:// if none found
-                return (urlSchemeRegex.test(value) ? '' : 'http://') + value;
+                return (urlSchemeRegex.test(value) ? '' : 'http://') + value.replace(/\s/g, '%20');
```

Comment:
Would it be worth trying to use `encodeURI` here to take care of all characters that need to be escaped, including spaces?